tests/functional/ubuntu/16.04/mon/initial_members/vagrant_variables.yml

```diff
@@ -0,0 +1,64 @@
+---
+
+# DEPLOY CONTAINERIZED DAEMONS
+docker: false
+
+# DEFINE THE NUMBER OF VMS TO RUN
+mon_vms: 1
```

Comment:
not for this scenario :) we only want to check ceph.conf generation for a monitor. 1 host is all we need here